pages/venue/online/index.js

```diff
@@ -0,0 +1,91 @@
+import CFSBanner from "../../../components/Speaker/banner";
+import EventDetails from "../../../components/Speaker/eventDetails";
+import Button from "../../../components/Buttons/button";
+import Link from "next/link";
+import CFPdata from "../../../config/cfp-data.json"
+export default function CallForSpeakers() {
+  return (
+    <div className="text-white mt-1">
+      <CFSBanner />
+      <div className="px-20 py-12">
+            <div>
+
+            <h2 className="text-2xl font-bold mb-8 ">Hello! Thank you for your interest in speaking at AACoT&apos;24 Online Edition</h2>
+            <div className="text-lg mb-4 lg:text-base">
+             The AsyncAPI Conf on Tour is an official event created by the AsyncAPI Initiative. This conference is aimed primarily at the community to share and exchange experiences between existing users and new members. We aim to integrate new members into the community and expand their knowledge about the project.
+            </div>
+
+  {/*<div className="text-lg mb-14 lg:mb-10 lg:text-base">
+            AACoT Madrid Edition will be hosted by <a href="https://www.ibm.com/" className="font-semibold text-blue-400">IBM</a>, one of the leading digital transformation companies in Spain. The conference will take place at one of IBM&#39;s offices in London, providing attendees with an opportunity to network with industry leaders and peers while exploring the city&#39;s vibrant culture. During the conference, attendees can expect to hear from industry experts including Fran Mendez, who will give a keynote on the history of AsyncAPI.
+            </div>*/}
```

Comment:
Plan on using this?

Reply:
Those commented parts where added by @thulieblack.
So shall I remove them ma'am?

Reply:
Yes keeping it for future use